content/blogs/2022-11-19-create-data-pipeline-bigquery-google-cloud.md

```diff
@@ -170,11 +166,10 @@ curl -v "<http://localhost:8080/api/v1/executions/trigger/io.kestra.gcp/extract-
     -F destinationFile="analysis_data_18"
 ```
 
-### Storage Write
+### Storage Write ###
+The [StorageWrite](../plugins/plugin-gcp/tasks/bigquery/io.kestra.plugin.gcp.bigquery.StorageWrite.md) plugin, is used for importing data from diverse sources like databases, NoSQL, queues, or other plugins into BigQuery. This is especially useful when data is stored externally. Another advantage of using this plugin is to avoid quotas limitation to ingest the data while data streaming in real-time or in batch job writing.
```

Comment:
```suggestion
The [StorageWrite](../plugins/plugin-gcp/tasks/bigquery/io.kestra.plugin.gcp.bigquery.StorageWrite.md) plugin is used for importing data from various sources, including databases, distributed message queues, or other plugins, into BigQuery. This is especially useful when data is stored externally. Another advantage of using this plugin is to avoid quotas limitation during streaming and batch data ingestion.
```